api-catalog-ui/frontend/package.json

```diff
@@ -18,7 +18,7 @@
         "loglevel": "1.6.1",
         "mineral-ui": "0.58.0",
         "mineral-ui-icons": "0.5.0",
-        "npm": "7.5.4",
+        "npm": "^7.20.5",
```

Comment:
Please remove the ^ to keep the stable version

Reply:
Yep noticed it, I was about to fix it already